### ISSUE TITLE
All Posts hyperlink on homepage not descriptive enough

### ISSUE THREAD

**Issue by cswitenky** [cswitenky/cswitenky#2]
<img width="1307" alt="image" src="https://github.com/cswitenky/cswitenky/assets/23012725/20371727-3445-480f-8aea-565a2e5cf5a1">

The 'See all posts [here](https://switenky.com/posts).' string on the [homepage](https://switenky.com) is not descriptive enough for screen reader users. It should be '[See all posts](https://switenky.com/posts)' or something similar.

Source: [PageSpeed Insights](https://pagespeed.web.dev/analysis/https-switenky-com/orn9rwxga7)
